<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanOpenTestDlg</class>
 <widget class="QWidget" name="CanOpenTestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CanTest</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Infomation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditCardName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>RX_Buffer</string>
        </property>
        <property name="buddy">
         <cstring>lineEditRXBuffer</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelTXCom">
        <property name="text">
         <string>TX_Complete</string>
        </property>
        <property name="buddy">
         <cstring>lineEditTXCom</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>OverRun</string>
        </property>
        <property name="buddy">
         <cstring>lineEditOverRun</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditRXBuffer">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditOverRun">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditRX">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Digit Outputs</string>
        </property>
        <property name="buddy">
         <cstring>lineEditDO</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditTX">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Digit Inputs</string>
        </property>
        <property name="buddy">
         <cstring>lineEditDI</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditTXCom">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>RX</string>
        </property>
        <property name="buddy">
         <cstring>lineEditRX</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEditDI">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Error</string>
        </property>
        <property name="buddy">
         <cstring>lineEditError</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEditDO">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditTXBuffer">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CardCount</string>
        </property>
        <property name="buddy">
         <cstring>lineEditCardCount</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditError">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TX</string>
        </property>
        <property name="buddy">
         <cstring>lineEditTX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditCardCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>TX_Buffer</string>
        </property>
        <property name="buddy">
         <cstring>lineEditTXBuffer</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Bus</string>
        </property>
        <property name="buddy">
         <cstring>lineEditBus</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="lineEditAI">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>COB-ID</string>
        </property>
        <property name="buddy">
         <cstring>lineEditNodeID</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Analog Inputs</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAI</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEditNodeID">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CardName</string>
        </property>
        <property name="buddy">
         <cstring>lineEditCardName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditBus">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="lineEditAO">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Analog Outputs</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAO</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelMsgRev">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Operation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="runPushButton">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopPushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>ID</string>
            </property>
            <property name="buddy">
             <cstring>lineEditID</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditID">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Data</string>
            </property>
            <property name="buddy">
             <cstring>lineEditData</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditData"/>
          </item>
          <item>
           <widget class="QPushButton" name="transmitPushButton">
            <property name="text">
             <string>Transmit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditCardCount</tabstop>
  <tabstop>lineEditCardName</tabstop>
  <tabstop>lineEditBus</tabstop>
  <tabstop>lineEditError</tabstop>
  <tabstop>lineEditRX</tabstop>
  <tabstop>lineEditTX</tabstop>
  <tabstop>lineEditTXCom</tabstop>
  <tabstop>lineEditTXBuffer</tabstop>
  <tabstop>lineEditRXBuffer</tabstop>
  <tabstop>lineEditOverRun</tabstop>
  <tabstop>lineEditNodeID</tabstop>
  <tabstop>lineEditDI</tabstop>
  <tabstop>lineEditDO</tabstop>
  <tabstop>lineEditAI</tabstop>
  <tabstop>lineEditAO</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>runPushButton</tabstop>
  <tabstop>stopPushButton</tabstop>
  <tabstop>lineEditID</tabstop>
  <tabstop>lineEditData</tabstop>
  <tabstop>transmitPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
